<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loadImgBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color: #d35400;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;</string>
    </property>
    <property name="text">
     <string>Load image</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstImage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>371</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #dddddd;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>22</width>
      <height>521</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="secondImage">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>371</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #dddddd;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="reduceColorsSlider">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>22</width>
      <height>521</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>660</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>690</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>660</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>669</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>694</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reduceColorsSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>703</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>727</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>705</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
